### DISCUSSION TITLE
Is there a way to use external source to populate MultiTextField field with picker editor?

### DISCUSSION THREAD

**Discussion by MikeAlhayek** [OrchardCMS/OrchardCore#9058] [Q&A] [answered]
I created a migration that would create part using fields. One of the fields is a `MultiTextField` with `picker` editor. But in my case, I want the picker to be read only "the user cannot add values." I also want the picker values to pulled from an external source "API"

I am basically trying to create "autocomplete" like field that would complete using an API. How can I make the "Picker" editor read only so the user can't add new values? Also, how can I pull the values from an external API?

**Comment by vitalybrandes** [comment, marked as answer]
If i get right, all this are in ADMIN area?! 
Sounds like you need own editor for this
and maybe regular textfiled will be enough.
